config/config.go

```diff
@@ -971,6 +949,20 @@ func (c *Config) CheckExchangeConfigValues() error {
 	return nil
 }
 
+// CheckBankAccountConfig checks all bank accounts to see if they are valid
+func (c *Config) CheckBankAccountConfig() {
```

Comment:
Okay so i got this to a workable point that isn't annoying to use
It will by default search for global level bank accounts and then exchange level bank accounts if a global one isn't found
if neither are found it will just error out as normal

Is  this an acceptable workaround @gloriousCode

Reply:
Yes 🎉